config/output_schemas/intram_uk_itl2.toml

```diff
@@ -0,0 +1,11 @@
+["Area Code (ITL3)"]
```

Comment:
Should that be "Area Code(ITL2)" rather than ITL3?

Reply:
Totally! Corrected.